.changeset/unlucky-walls-tie.md

```diff
@@ -1,5 +1,7 @@
 ---
-"@coinbase/onchainkit": minor
```

Comment:
@0xAlec please be careful when you write the changeset, in introducing the word `minor`, I know you fix it now. But this kind of typo can be very risky.